mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs

```diff
@@ -339,7 +354,13 @@ pub fn utxo_asset_docker_node<'a>(docker: &'a Cli, ticker: &'static str, port: u
 
 pub fn geth_docker_node<'a>(docker: &'a Cli, ticker: &'static str, port: u16) -> DockerNode<'a> {
     let image = GenericImage::new(GETH_DOCKER_IMAGE, "stable");
-    let args = vec!["--dev".into(), "--http".into(), "--http.addr=0.0.0.0".into()];
+    let args = vec![
+        "--dev".into(),
+        "--http".into(),
+        "--http.addr=0.0.0.0".into(),
+        "--dev.period=0".into(),
+        "--verbosity=4".into(),
```

Comment:
oh, this is a leftover from attempts to fix Geth nft tests.

I was changing seconds between new blocks creation with `dev.period` flag.
In this commit https://github.com/KomodoPlatform/komodo-defi-framework/pull/2100/commits/03860e100be3b7fb134537f2e6b74cb8891878b2 decided to remove new args, since `dev.period=0` is actually a default Geth falg (node mines blocks when there is the new transaction).
And logs verbosity 4 level has just some additional messages for debugging, while we dont actually use Geth log file in CI. I suppose the default `--verbosity 3` value is enough.